<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewRepository</class>
 <widget class="QDialog" name="AddNewRepository">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>315</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>315</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>315</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add New Repository</string>
  </property>
  <widget class="QLineEdit" name="Edit_URL">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_Login">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ComboBox_RepositoryType">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Password">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>138</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_Password">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="ComboBox_BranchName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Label_URL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>77</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>URL:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Add_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Add and Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Login">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>107</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Login:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_ChooseRepositoryType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose the type of repository:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Connect">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>165</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_MainString">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the URL, login, and password of your repository:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Select">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the branch which you want to follow:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_CheckNewCommits">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select interval for checking the new commits:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ComboBox_TimeInterval">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Cancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Canc&amp;el</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ComboBox_RepositoryType</tabstop>
  <tabstop>Edit_URL</tabstop>
  <tabstop>Edit_Login</tabstop>
  <tabstop>Edit_Password</tabstop>
  <tabstop>Button_Connect</tabstop>
  <tabstop>ComboBox_BranchName</tabstop>
  <tabstop>ComboBox_TimeInterval</tabstop>
  <tabstop>Button_Add_Save</tabstop>
  <tabstop>Button_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
